docs/installation.md

```diff
@@ -20,6 +20,69 @@ You can also spin up one of our pre-configured Cloud-based dev environments:
 
 Have a look at the [Lingua Franca playground](https://github.com/lf-lang/playground-lingua-franca) for more details.
 
+## Compiler
+
+There are multiple options available for installing the Lingua Franca compiler and other command line tools, as listed below. See the [handbook](./tools/command-line-tools.mdx) for usage instructions.
+
+### Install Script
+
+Run the following command in your terminal to install the latest release (on Windows, use WSL):
+
+```
+curl -Ls https://install.lf-lang.org | bash -s cli
+```
+
+You can also install the nightly pre-release:
+
+```
+curl -Ls https://install.lf-lang.org | bash -s cli nightly
+```
+
+You can use the `--prefix=<path>` argument to change the default install location.
+
+### AUR
+
+There are binary packages available in the Arch user repository, which you can install using your favorite AUR helper. For instance, with `yay`:
+
+```
+yay -S lf-cli-bin
+```
+
+or for the nightly pre-release:
+
+```
+yay -S lf-cli-nightly-bin
+```
+
+### Nixos
+
+LFC is packaged in [nixpkgs](https://github.com/NixOS/nixpkgs/blob/nixos-23.11/pkgs/development/compilers/lingua-franca/default.nix#L28) and is available via the binary cache.
+
+Run
+```
+ nix shell nixpkgs#lingua-franca
+```
+to temporarily install lfc in your current shell environment.
+
+### Manual Download
+
+Regular and nightly release builds of the command line tools can be downloaded from the [Lingua Franca release page](https://github.com/lf-lang/lingua-franca/releases). Download the archive that matches your OS and architecture, and extract the contents.
+
+### From Source
+
+Please refer to the [Lingua Franca GitHub repository](https://github.com/lf-lang/lingua-franca) for build instructions.
+
+If you'd like to contribute to Lingua Franca, you can find details about the recommended developer setup here.
+
+## Lingo the Lingua-Franca Package-Manager
+
+Lingo is the package-manager and build tool of the lingua-franca ecosystem.
+
+```
+    cargo install lingua-franca
+    PATH = $PATH:~/.cargo/bin
```

Comment:
Shouldn't this be an export command?

Reply:
the export is not necessary ;D it also works without it

Reply:
Ok, right. It doen't work with the spaces around the `=` though.
```suggestion
    PATH=$PATH:~/.cargo/bin
```